os/arch/arm/src/armv7-m/up_assert.c

```diff
@@ -271,6 +271,16 @@ static void up_dumpstate(void)
 	/* Show interrupt stack info */
 
 	lldbg("sp:     %08x\n", sp);
+	if (current_regs) {
+		lldbg("*******************************************\n");
+		lldbg("Asserted during Interrupt mode\n");
+		lldbg("*******************************************\n");
+		lldbg("IRQ num: %d \n", getipsr());
+		lldbg("IRQ handler: %08x \n", g_irqvector[getipsr()].handler);
+#ifdef CONFIG_DEBUG_IRQ_INFO
+		lldbg("IRQ handler: %s \n", g_irqvector[getipsr()].irq_name);
```

Comment:
Usually we don't enable this config so that TRAP should convert the address to the symbol. Could you check TRAP for this?

Reply:
Updated